{{cookiecutter.repo_name}}/tox.ini

```diff
@@ -1,9 +1,11 @@
 [tox]
 envlist ={% if '1.8' in cookiecutter.django_versions %}
-    {py27,py33,py34,py35}-django-18{% endif %}{% if '1.9' in cookiecutter.django_versions %}
-    {py27,py34,py35}-django-19{% endif %}{% if '1.10' in cookiecutter.django_versions %}
-    {py27,py34,py35}-django-110{% endif %}{% if 'master' in cookiecutter.django_versions %}
-    {py27,py34,py35}-django-master{% endif %}
+    {py27,py33,py34,py35,py36}-django-18{% endif %}{% if '1.9' in cookiecutter.django_versions %}
+    {py27,py34,py35,py36}-django-19{% endif %}{% if '1.10' in cookiecutter.django_versions %}
+    {py27,py34,py35,py36}-django-110{% endif %}{% if '1.11' in cookiecutter.django_versions %}
```

Comment:
Although that is probably working, Python 3.6 [isn't listed in the versions](https://docs.djangoproject.com/en/2.0/faq/install/#what-python-version-can-i-use-with-django) usable with Django < 1.11.

Reply:
I've removed that now